### PR TITLE
add load data from yuque

### PR DESCRIPTION
**Pull request by yyz629**
Complete load docs from yuque.

### CODE REVIEW

**Comment by yyz629** [comment]
The annotation issue has been modified, please check it again.

**Review by hwchase17 — APPROVED** [review]
im happy to fix typing - could you add an example notebook? thanks

File: langchain/document_loaders/yuqueloader.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+from typing import List, Tuple, Dict, Union
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+
+class YUQUELoader(BaseLoader):
```
Comment (leo-gan) [inline langchain/document_loaders/yuqueloader.py:7]: Is there a reason to use all capital for the Class name (`YUQUELoader`)? 
`YuqueLoader` would be a better fit for the standard naming convention.

**Comment by leo-gan** [comment]
@yyz629 Hi , could you, please, resolve the merging issues and address the last comments (if needed)? After that, ping me and I push this PR for the review. Thanks!

**Comment by efriis** [comment]
Closing because the PR wouldn't line up with the current directory structure of the library (would need to be in /libs/langchain/langchain instead of /langchain). Feel free to reopen against the current head if it's still relevant!

**Pull request closed without merging**